noDefaultMetrics.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Prometheus client
+ * @module Prometheus client
+ */
+
+'use strict';
+
+exports.register = require('./lib/register');
+exports.contentType = require('./lib/register').contentType;
+
+exports.Counter = require('./lib/counter');
+exports.Gauge = require('./lib/gauge');
+exports.Histogram = require('./lib/histogram');
+exports.Summary = require('./lib/summary');
+exports.Pushgateway = require('./lib/pushgateway');
+
+exports.linearBuckets = require('./lib/bucketGenerators').linearBuckets;
+exports.exponentialBuckets = require('./lib/bucketGenerators').exponentialBuckets;
```

Comment:
@voxpelli yeah, I've wrote it in first comment as an option, but it will be breaking change. As for me both options will be ok :)